nextflow.config

```diff
@@ -74,6 +75,11 @@ params {
     quantification_min_peak_width    = 0.2
     quantification_mapping_tolerance = 0
 
+    // Epicore settings
+    epicore_min_overlap                      = '11'
+    epicore_max_step_size                    = '5'
+    epicore_min_epi_length                   = '11'
```

Comment:
```suggestion
    epicore_min_overlap    = '11'
    epicore_max_step_size  = '5'
    epicore_min_epi_length = '11'
```

Reply:
Also, are those integers or strings? Coul be 11 instead of '11'?